lib/siilar/client/clients.rb

```diff
@@ -13,9 +13,12 @@ def tags
       @services[:tags] ||= Client::TagsService.new(self)
     end
 
+    def tracks
+      @services[:tracks] ||= Client::TracksService.new(self)
+    end
+
     def users
       @services[:users] ||= Client::UsersService.new(self)
-    end
```

Comment:
Looks like this guy is the guilty one :cop: 
https://travis-ci.org/craftsmen/niland-siilar-ruby/builds/102544831
@gcolson could you fix it pls?